android/build.gradle

```diff
@@ -57,4 +58,5 @@ dependencies {
     androidTestImplementation "androidx.test.ext:junit:$androidxJunitVersion"
     androidTestImplementation "androidx.test.espresso:espresso-core:$androidxEspressoCoreVersion"
     implementation "io.intercom.android:intercom-sdk:$intercomSdkVersion"
+    implementation "com.google.firebase:firebase-messaging:$firebaseMessagingVersion"
```

Comment:
@jpike88 have you tried defining the firebase-messaging dep in your app's build.gradle instead ?